server/api_archive_integration_test.go

```diff
@@ -0,0 +1,275 @@
+package server
+
+import (
+	"archive/zip"
+	"bytes"
+	"fmt"
+	"io"
+	"math/rand"
+	"net/http"
+	"os"
+	"path/filepath"
+	"slices"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/glowlabs-org/gca-backend/glow"
+)
+
+func readFile(dir, name string) ([]byte, error) {
```

Comment:
I don't think this helper should be used. It's not adding enough convenience and it makes me think when I see it used that there are extenuating circumstances that require a separate function to be used.

Because `filepath.Join` has no error handling, it's not unweildly to just use it inline every time.

Reply:
generally in go you don't worry about verbosity and repetition too much. It's one of the big dividing aspects of the language - people who love go really love the simplicity and composability that comes with prioritizing readability through verbosity, and people who dislike go tend to highlight the verbosity and repetition as a detracting feature

But since we're writing go, we'll do it the go way :)

Reply:
When in Rome, do as the Romans do.

Or is it "When in Golang, do as the Golangers do" :)